guides/10_Video_Processor.md

```diff
@@ -0,0 +1,115 @@
+# Video Processing APIs
+
+## Introduction
+
+Amazon Chime SDK for JavaScript contains APIs to facilitate the processing of video stream and provides a simplified workflow for adding processing to the outgoing video feed.
+
+The APIs define a video processing stage as `VideoFrameProcessor`. A `VideoFrameProcessor` takes an array of `VideoFrameBuffer`s, applies the processing and outputs an array of `VideoFrameBuffer`s. Processors can be chained by their inputs and outputs. The outputs of each processor can be linked to the inputs of the next processor.
```

Comment:
Nit: 'APIs' isn't well defined (SDK API as a whole? Just a component of the API?).  Can probably just replace with `SDK defines`.

Reply:
Or 'APIs for video processing' like below